sqlalchemy_cockroachdb/requirements.py

```diff
@@ -181,7 +181,7 @@ def json_deserializer_binary(self):
         )
 
     def get_isolation_levels(self, config):
-        return {"default": "SERIALIZABLE", "supported": ["SERIALIZABLE", "AUTOCOMMIT"]}
+        return {"default": "SERIALIZABLE", "supported": ["SERIALIZABLE", "AUTOCOMMIT", "READ COMMITTED"]}
```

Comment:
question: do we need something that only allows READ COMMITTED if the CRDB version is v23.2 or greater?